docs/get-started/deploy-cloudfoundry-for-enterprise.md

```diff
@@ -0,0 +1,129 @@
+ 
+## Availability Set
+
+### Why
+
+For high availability of your Cloud Foundry infrastructure, you can utilize Availability Set for each of your resource pools.
+
+There are two types of Microsoft Azure platform events that can affect the availability of your virtual machines: planned maintenance and unplanned maintenance. To reduce the impact of downtime due to one or more of these events, we recommend configuring multiple virtual machines in an [Availability Set](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-manage-availability/) for redundancy.
+
+### How
+
+1. Open your manifest, and configure an availability set into your `resource_pools`.
+
+  ```
+  - name: resource_router
+    network: default
+    stemcell:
+      name: bosh-azure-hyperv-ubuntu-trusty-go_agent
+      version: '0000'
+    cloud_properties:
+      availability_set: <availability-set-name>
+      instance_type: Standard_D1
+  ```
+
+  If `<availability-set-name>` does not exist, it will be created automatically.
+
+2. Specify the resource_pool as `resource_router` in your job. Then these 2 VMs will be put into the availability set `<availability-set-name>`.
+
+  ```
+  - name: router_z1
+    instances: 2
+    resource_pool: resource_router
+    templates:
+    - {name: gorouter, release: cf}
+    - {name: metron_agent, release: cf}
+    networks:
+    - name: default
+      static_ips: [10.0.16.12, 10.0.16.22]
+    properties:
+      dropsonde: {enabled: true}
+  ```
+
+3. Deploy your Cloud Foundry.
+
+## Multiple Storage Account
+
+### Why
+
+Due to [Azure Storage Limits](https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/#storage-limits), the maximum number of highly utilized disks in one storage account is about 40 (20,000/500 IOPS per disk) for a Standard Tier VM. If you need high-performance, low-latency disk support, you can use [Azure Premium Storage](https://azure.microsoft.com/en-us/documentation/articles/storage-premium-storage-preview-portal/). For premium storage disks (P30), the maximum number of highly utilized disks in one storage account is about 31.
+
+If the number of disks in one storage account exceeds the maximum number, you need to split these disks into multiple storage accounts for performance optimizations.
+
+### How
+
+1. Make a plan for the mapping between the resource pools and the jobs.
+
+  For example, if your deployments need 80 runners (DEAs), then the number of disks is at least 80 which exceeds the maximum number in one storage account. So you need to configure at least 2 resource pools for runners.
+
+2. Specify an additional storage account in each of the resource pools.
+
+  Open your manifest, and configure `storage_account_name: <new-storage-account>` in your resource pool as follows. Then all the VMs in the pool will store their disks in the storage account `<new-storage-account>`.
+
+  ```
```

Comment:
Here is the suggested paragraph: 
By default there is one Standard Tier storage account created for each resource pool, providing 40 IOPS per disk based on Azure storage design(insert(link here). If you need higher performance, you need to create multiple storage accounts.  In addition, since each resource pool can only contain single storage account, when you need multiple storage accounts, you also need to split the resource pool into multiple resource pools, each with a single storage account. 
Note you can choose different types of storage accounts when considering performance optimization. For example, if you need high-performance, low-latency disk support, you can use [Azure Premium Storage](https://azure.microsoft.com/en-us/documentation/articles/storage-premium-storage-preview-portal/). For premium storage disks (P30), the maximum number of highly utilized disks per storage account is about 31.